pepper-apis/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -174,6 +174,10 @@ public static void clearAfterHandler() {
     }
 
     public static void main(String[] args) {
+        start(args, null);
+    }
+
+    public static void start(String[] args, SendGridSupplier sendGridSupplier) {
```

Comment:
Not too comfortable with the fact that I have to change real code just to facilitate testing. I'm hoping we can make Housekeeping easier to test in a more natural way when we seriously start refactoring it. For now, this change should be harmless.

Reply:
D.I.